schema/policy_test.go

```diff
@@ -421,7 +449,7 @@ var _ = Describe("Policies", func() {
 				It("should allow access be default", func() {
```

Comment:
b**y**, besides this TC is misleading: the test code does not verify if access is indeed granted - now it depends on "policy1". by the way: what is policy1? can we come up with a better name?